### ISSUE TITLE
Pyright does not understand NamedTuple unpacking

### ISSUE THREAD

**Issue by jmg-duarte** [microsoft/pyright#3021]
**Describe the bug**
Pyright reports a valid tuple unpacking as an error. 

**To Reproduce**
```python
from collections import namedtuple

T = namedtuple("T", ("a", "b", "c", "d"))

a, b, c, d = SetupResult(1, 2, 3, 4)
```

**Expected behavior**
No error was reported.

**VS Code extension or command-line**
VS Code extension

**Comment by erictraut** [comment]
This will be addressed in the next release.

**Comment by erictraut** [comment]
This is included in pyright 1.1.221, which I just published. It will also be included in the next release of pylance.
